amber-slsa-buildtype.json

```
@@ -0,0 +1,137 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Amber SLSA buildType",
+  "description": "Describes a SLSA buildType, as defined in https://slsa.dev/provenance/v0.2.",
+  "type": "object",
+  "required": [
+    "buildType",
+    "buildConfig",
+    "materials"
+  ],
+  "not": {
+    "required": [
+      "invocation"
+    ]
+  },
+  "properties": {
+    "buildType": {
+      "const": "https://github.com/project-oak/transparent-release/amber-slsa-buildtype.json"
+    },
+    "buildConfig": {
+      "type": "object",
+      "properties": {
+        "command": {
+          "type": "string",
+          "description": "Command passed to the builder image.",
+          "examples": [
+            "./scripts/runner build-functions-server"
+          ]
+        },
+        "outputPath": {
+          "type": "string",
+          "description": "The path, relative to the root of the git repository, where the artifact built by the command is expected to be found.",
+          "examples": [
+            "./oak_functions/loader/bin/oak_functions_loader"
+          ]
+        },
+        "expectedArtifactDigest": {
+          "type": "object",
+          "description": "Expected hash of the output artifact. Can be ommitted.",
```

```suggestion
          "description": "Expected hash of the output artifact. Optional.",
```

nit: I think calling such fields optional is more conventional.